src/data/api.js

```diff
@@ -0,0 +1,16 @@
+const ID = 'I0ugL4x2K9pTkncbrX8V';
+
+async function add(newScore) {
+  await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${ID}/scores/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(newScore),
+  })
+    .then((ele) => ele.json());
+}
+async function get() {
+  return fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${ID}/scores/`);
+}
```

Comment:
- [ ] Kindly change all the functions in your code to the arrow function instead of the keyword function. This is part of the project requirements. Take a look please :point_down: 

![project-req](https://user-images.githubusercontent.com/26597432/202419490-f27aeaad-d457-4276-8a63-cdc6cc08c805.png)